cli/cli/Services/BeamoLocalSystem_Docker.cs

```diff
@@ -305,6 +304,7 @@ public async Task<string> PrepareBeamoServiceImage(BeamoServiceDefinition servic
 			{
 				BeamableLogger.LogError(ex);
 				BeamableLogger.LogError(ex?.InnerException);
+				throw new CliException($"Failed while building image = [{imageName}]");

```

Comment:
it would be nice to include some more information on the `CliException` from the inner `ex` itself.

Reply:
Going to do it, thanks